plugins/kubernetes-backend/src/service/getKubernetesObjectsByServiceIdHandler.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { KubernetesFetcher } from './KubernetesFetcher';
+import { KubernetesClusterLocator } from '../cluster-locator/types';
+import { Logger } from 'winston';
+import { ObjectsByServiceIdResponse } from './types';
+
+export type GetKubernetesObjectsByServiceIdHandler = (
```

Comment:
Would be nice to have a comment on these public interfaces, what their role is in the system, etc. 😅 

Let's also move them to a common `types.ts` so that it's a bit easier to get an overview of the moving parts, and not have them reside next to specific implementations.